service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Model\Entity;
+
+use Application\Validators\LPAValidator;
+use Laminas\Form\Annotation;
+use Laminas\Form\Annotation\AttributeBuilder;
+use Laminas\Form\Annotation\Validator;
+use Laminas\Validator\InArray;
+use Laminas\Validator\NotEmpty;
+use Laminas\Validator\Regex;
+use JsonSerializable;
+
+/**
+ * DTO for holding data required to make new case entry post
+ * @psalm-suppress MissingConstructor
+ * Needed here due to false positive from Laminas’s uninitialised properties
+ */
+class CaseData implements JsonSerializable
+{
+    #[Validator(InArray::class, options: ["haystack" => ['passport', 'drivinglicense', 'nino']])]
+    private string $verifyMethod;
```

Comment:
I don't think we should record this: we won't know it until at the point the case is created, and the user might change or select multiple options.

We'll need to store any verified documents on the record, but that will come in a later request